### DISCUSSION TITLE
Looking for YOUR performances to ship with MiniDexed

### DISCUSSION THREAD

**Discussion by probonopd** [probonopd/MiniDexed#325] [General]
Let's give MiniDexed some iconic DX/TX 80s/90s era performances out of the box, making good use of all 8 Tone Generators, detune, stereo shifting, and reverb.

Have some to share? Just zip your performance files and attach them to this thread, ideally with some description and how you would like to be credited.

Some inspriration:
* [Jim Daneker: DX Dreams Walkthrough - Recreating Iconic 80s DX7 Sounds](https://youtu.be/bkQ9z6Q77Z8?t=300) (e.g., around minute 6:20)
* [Espen Kraft: FM Monster Duel! Yamaha TX816 vs TX802](https://www.youtube.com/watch?v=TutoLkJ_bks)

**Comment by warpeggio** [comment]
These don't use all 8 TGs, but they do sound great. I think as long as we attribute them to A BEHR we would be okay, but it might also be worth reaching out for explicit permission.

https://www.youtube.com/watch?v=QzILHOBp1zU

**Comment by Banana71** [comment]
Well then I'll make a start:

003_SadStrings
004_ModernE-Piano
005_Mama
006_Weekend
007_Floating
008_RockOrgan

[performance.zip](https://github.com/probonopd/MiniDexed/files/9742298/performance.zip)

**Reply by Banana71** [reply]
Some new Performances:

000001_VintageE-Piano.ini
000002_ModernE-Piano.ini
000003_SadStrings.ini
000004_Pad-1.ini
000005_Floating.ini
000006_Orchester.ini
000007_RockOrgan.ini
000008_Mama.ini
000009_Weekend.ini
000010_SynTh.ini

[Performance Banana 221016 .zip](https://github.com/probonopd/MiniDexed/files/9793698/Performance.Banana.221016.zip)

https://user-images.githubusercontent.com/104296205/196020844-9fa62624-af87-444b-b141-85d6c81cf21d.mp4

**Reply by Banana71** [reply]
Performance update:

000011_HeraStrings
000012_Cassini
000013_SynStrings
000014_Queen-WW2LF
000015_CP-70Cho
000016_AmpUpright 1
000017_Orchester 2
000018_Barock

[Performance Banana 11_18.zip](https://github.com/probonopd/MiniDexed/files/9843736/Performance.Banana.11_18.zip)


https://user-images.githubusercontent.com/104296205/197351866-70e0aea0-1f3f-47c0-97dc-9b8120721915.mp4

**Reply by Banana71** [reply]
### Piano Library

000019_GrandPiano.ini
000020_Chinematic.ini
000021_GrandLea.ini
000022_HonkyTonk.ini
+
000023_Mermaids.ini²

Authentic FM sound quality. Using the latest sampling technology.
Needs less then 1 GB Sampling Ram
 
### **System requirements**

_**Operating system**: Raspberry Pi , MiniDexed
**Hard disk free space**: 32 kB
**Download**_: [Piano Library 19-22.zip](https://github.com/probonopd/MiniDexed/files/9845926/Piano.Library.19-22.zip)
_**Update**_: [Piano Library 19-22 +.zip V1.1](https://github.com/probonopd/MiniDexed/files/9851180/Piano.Library.19-22.%2B.zip) fixes some volume issues
²  Mermaids and whales sing to Poseidon on the ancient shores of Atlantis. Sample from the new Choir Library (Release Nov'22)
_*please forget to register_

https://user-images.githubusercontent.com/104296205/197524034-52c4b125-7b59-468f-a1da-77f416d6bc6f.mp4

Attention! Some whales swam too close to the microphone

**Comment by pik33** [comment]
Excellent set of sounds. Mermaids and Grand Piano - wow! 

Edit:

> Authentic FM sound quality. Using the latest sampling technology.
> Needs less then 1 GB Sampling Ram

ROTFL ! :)

**Comment by Banana71** [comment]
### Banana71 Library Complete, so far

**_Download_**: [Banana71 - 36 Performances.zip](https://github.com/probonopd/MiniDexed/files/9896595/Banana71.-.36.Performances.zip)

https://user-images.githubusercontent.com/104296205/198896415-e6f6575c-c94f-448c-bd02-b1678758c853.mp4


https://user-images.githubusercontent.com/104296205/198896429-43527823-8de7-4f5d-8a3c-152c73230df1.mp4



</HEAD>

<BODY TEXT="#000000">
<TABLE FRAME=VOID CELLSPACING=0 COLS=3 RULES=NONE BORDER=0>
	<COLGROUP><COL WIDTH=156><COL WIDTH=156><COL WIDTH=156></COLGROUP>
	<TBODY>
		<TR>
			<TD WIDTH=156 HEIGHT=18 ALIGN=LEFT>000001_GrandPiano</TD>
			<TD WIDTH=156 ALIGN=LEFT>000019_NewAge EP</TD>
			<TD WIDTH=156 ALIGN=LEFT>000050_Mermaids_B²</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000002_Cinematic</TD>
			<TD ALIGN=LEFT>000032_SadStrings</TD>
			<TD ALIGN=LEFT>000051_Sacred_A</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000003_GrandLea</TD>
			<TD ALIGN=LEFT>000033_Pad-1</TD>
			<TD ALIGN=LEFT>000052_Sacred_B</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000004_Phil's CP-70</TD>
			<TD ALIGN=LEFT>000034_Floating</TD>
			<TD ALIGN=LEFT>000053_Mama</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000005_DreamPiano</TD>
			<TD ALIGN=LEFT>000035_Orchester</TD>
			<TD ALIGN=LEFT>000054_Choir 1TG_Dry</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000006_BrightPiano</TD>
			<TD ALIGN=LEFT>000036_HeraStrings</TD>
			<TD ALIGN=LEFT>000064_RockOrgan</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000007_NewAge AP</TD>
			<TD ALIGN=LEFT>000037_Orchester 2</TD>
			<TD ALIGN=LEFT>000065_Queen-WW2LF</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000008_AmpUpright</TD>
			<TD ALIGN=LEFT>000038_SQR-BassPad</TD>
			<TD ALIGN=LEFT>000080_Weekend</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000009_HonkyTonk</TD>
			<TD ALIGN=LEFT>000039_SynStrings</TD>
			<TD ALIGN=LEFT>000081_SynTh</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000016_VintageE-Piano</TD>
			<TD ALIGN=LEFT>000040_Barock</TD>
			<TD ALIGN=LEFT>000082_Cassini</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000017_ModernE-Piano</TD>
			<TD ALIGN=LEFT>000048_Choir</TD>
			<TD ALIGN=LEFT>000127_INIT</TD>
		</TR>
		<TR>
			<TD HEIGHT=18 ALIGN=LEFT>000018_CP-70Cho</TD>
			<TD ALIGN=LEFT>000049_Mermaids_A</TD>
			<TD ALIGN=LEFT><BR></TD>
		</TR>
	</TBODY>
</TABLE>
</BODY>

</HTML>
²Designed for 88 Keys Masterkeyboard

**Reply by probonopd** [reply]
Great work @Banana71, now I am curious - what is your workflow for making these?

**Reply by Banana71** [reply]
Sometimes i follow an idea and have a defined goal or i adapt existing sounds to my needs.  
I program my sounds on the PC with DEXED, then i create the performances on the MiniDexed. 
The fine-tuning i do directly on the device. (FX, Envelope, LFO)
Otherwise more than 30 years of experience with FM synthesizers (DX7 II-FD, SY-99, MODX-8) help.

My current goal is a MiniDexed powered digital piano. 
Piano, E-Piano, Pads, Choir, Strings and Organs are in my focus. 
That's why I haven't created any MultichannelSounds and so rar SynthesizerSounds yet.

**Reply by probonopd** [reply]
> Otherwise more than 30 years of experience with FM synthesizers (DX7 II-FD, SY-99, MODX-8) help.

It shows! :1st_place_medal:

**Reply by dcoredump** [reply]
Many thanks @Banana71!

(Stupid?) question: What are the base SYSEX files for your performances?

**Reply by Banana71** [reply]
Hi dcoredump,

i downloaded on [www.yamahablackboxes.com](url) the original Factory - and Cardridge VRC - SYSEX from DX7 and DX7 II
and i created two Banks with mostly own or modified Patches. (Download will follow soon)

When i use the original Yamaha Sounds most of time i spent for adjusting the Operator Level to optimize the Velocity dynamic. ;-)
[https://github.com/probonopd/MiniDexed/discussions/352](url)

**Reply by Kirtai** [reply]
> My current goal is a MiniDexed powered digital piano. Piano, E-Piano, Pads, Choir, Strings and Organs are in my focus.

Oh wow, that sounds _amazing_. I've been looking into expanding my digital piano and this would be a great addition :)

**Comment by ghost** [comment]
Where should the performance .ini files be placed in the directory structure?

**Reply by Banana71** [reply]
Hi ImaNottellinya,
i found the following explanation:

https://github.com/probonopd/MiniDexed/pull/228#issuecomment-1133820616

**Reply by Banana71** [reply]
@probonopd i edited the Wiki and i wrote some words to Files/performance.ini. Can you please proofread and correct accordingly. My language skills are not sufficient.

**Reply by probonopd** [reply]
Thanks @Banana71.

**Comment by ghost** [comment]
Peter

Many thanks.

Ima

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by Banana71** [comment]
My currently favorite MiniDexed sound

[BananaSYSEX&Strings.zip](https://github.com/probonopd/MiniDexed/files/10081609/BananaSYSEX.Strings.zip)

/performance/000041_BananaStrings.ini
/sysex/voice/000001_BanaROM71A.syx
/sysex/voice/000002_BanaROM71B.syx

https://user-images.githubusercontent.com/104296205/203718065-1efd512c-e1a8-476d-86ff-ef5e8034a8a4.mp4

**Reply by probonopd** [reply]
Very cool! Thank you. :+1:

**Reply by probonopd** [reply]
@Banana71 those have the same numbers as performances in "Banana71 Library Complete, so far". 
* Should we replace the performances with the same number?
* If not, please suggest other numbers for them.
Thanks!

**Reply by Banana71** [reply]
What a honour,
I think 20 of my performances are enough.  there was too many performances of the same kind and i missed some new :-). 
The numbering starts from 65 so that everyone can sort their own performances more easily

[performance.zip](https://github.com/probonopd/MiniDexed/files/10201691/performance.zip)

<html>

<body>


65_GrandPiano.ini | 75_HeraStrings.ini
-- | --
66_Cinematic.ini | 76_Barock.ini
67_GrandLea.ini | 77_BananaStrings.ini
68_Phil's CP-70.ini | 78_Mermaids_B.ini
69_NewAge AP.ini | 79_Sacred_A.ini
70_ElectricPiano.ini | 80_Mama.ini
71_CP-70Cho.ini | 81_Choir 2.ini
72_NewAge EP.ini | 82_RockOrgan.ini
73_SadStrings.ini | 83_SynTh.ini
74_Pad-1.ini | 84_FM-Drumkit 1.ini


</body>

</html>

**Reply by pik33** [reply]
I don't understand this number problem. There are these 36 performances in this: [Banana71 - 36 Performances.zip](https://github.com/probonopd/MiniDexed/files/9896595/Banana71.-.36.Performances.zip). What was added after this is FM Drumkit at 84 and BananaStrings at 41. There were no number collisions with these 2 new performances.

 Now, there is a new zip with  new numbers and the same names - why?

--------------

I think the best performance numbering method is to use General Midi type "instrument blocks" (pianos at 0, organs at 16, etc.)

**Reply by Banana71** [reply]
@pik33
We are looking for the Performances that should be included in the main download. (a kind of factory sound for MiniDexed)
The numbering from 65 offers the advantage that you can sort your own performances directly to the front.
My programmed performances is constantly growing, so that it is slowly becoming confusing here. Maybe I should delete some of my post and only update the files in my first Post.

**Reply by probonopd** [reply]
@Banana71 Your sounds are great, please don't delete any of them. But I noticed that sometimes you are using the same number for different sounds. Why?

**Reply by pik33** [reply]
So we need a higher menu/directory level - a bank of performances.
Select bank->select performance - so there can be thousands of them on a SD and you can still find something.

**Reply by Banana71** [reply]
> So we need a higher menu/directory level - a bank of performances. Select bank->select performance - so there can be thousands of them on a SD and you can still find something.

Yes, that would make things easier.

**Comment by probonopd** [comment]
@Banana71 how would you go about making something that sounds like the "sparkling" in the first ~4 seconds of https://www.youtube.com/watch?v=nHlOhv2ow1Y?

**Reply by Banana71** [reply]
@probonopd , i have modified the Weekend performance.
hold the sustain pedal and play single notes in C minor, for example
I can't record audio right now.

[000128_Feenstaub.zip](https://github.com/probonopd/MiniDexed/files/10211814/000128_Feenstaub.zip)

**Reply by probonopd** [reply]
Thank you very much @Banana71. You are a true master of the subject. Nothing can match 30 years of experience with FM! :+1:

**Comment by mongio** [comment]
sorry for being so disinformed on the matter but: the patches used in the performances are intended to be downloaded with getsysex.sh? Can I have some help to use getsysex? I only have a windows pc. Thank you!

**Reply by probonopd** [reply]
In the latest build, @Banana71's performances come bundled with MiniDexed.
Since they include their ow voice data inside the files, they work even without downloading any voices separately.

If you want to download the voice data in an automated way, you currently need Linux or BSD. For Windows, you'd either need to download the files in `getsysex.sh` by hand, or would need to write a `.bat` or PowerShell script, I guess. (I don't use Windows.)

**Reply by mongio** [reply]
oh, thanks, it's all clear now and thanks @Banana71 too:-)

**Comment by Kirtai** [comment]
Someone on reddit produced a nice harp sound. Could be useful.

See [here](https://www.reddit.com/r/synthesizers/comments/zp8jef/i_made_a_magical_harp_patch_on_dx7/)

There's a sysex available [here](https://drive.google.com/file/d/1uC5mqN6aWWbbJslhS28YPpW4hEUfm01p/view?usp=share_link) made by the creator.

**Reply by Banana71** [reply]
Hi @Kirtai,

here is the MiniDexed Version of MagicalHarp you shared with us. Thx to sexzlizardbrain from reddit.com who create this sound with the Yamaha DX7.
In this version the sound is fourfold layered. Purists should switch off the MidiChannel in TG2 - TG4.
The magic in this sound comes from its realese time. When the note is held, the sound dies out faster than when notes are played briefly.

[000096_MagicalHarp.zip](https://github.com/probonopd/MiniDexed/files/10363303/000096_MagicalHarp.zip)

Peter

**Reply by Kirtai** [reply]
Thank you :)

**Comment by Banana71** [comment]
Multipart Performance with short Demo:

[000101_Pattern 1.zip](https://github.com/probonopd/MiniDexed/files/10547951/000101_Pattern.1.zip)

https://user-images.githubusercontent.com/104296205/215801981-6ea64af2-08a0-41c6-9c4a-85f5a33ca345.mp4

**Comment by Banana71** [comment]
Pattern 2
a small session in cubase inspired by Nina Chuba - Ich hass dich (German Pop)
All sounds come from the miniDexed. No EQ, compressor or other effect.


https://user-images.githubusercontent.com/104296205/221351854-663fcd1a-b194-4141-a277-b893ab2a710e.mp4

**Reply by probonopd** [reply]
Makes me wonder whether we should provide some MIDI files for our users to test MiniDexed with. Maybe we could even squeeze a MIDI player into MiniDexed... so that it could demonstrate its power out of the box

**Reply by Banana71** [reply]
To avoid misunderstandings, this demo pattern 2 was not created out of the box. This required several instances of miniDexed. Each track was created from its own performance. If you put them together cleverly, you would still need 16 TG's.  Resulting 2 miniDexed.
I like the idea of demo midi files.

![DAW - Ich hass Dich](https://user-images.githubusercontent.com/104296205/221371573-2204d87b-1886-4ab8-bc06-471456d3fd23.jpg)

Drums and Pad are well-known Performances (FM-Drumkit 1 and Orchestra 2) I provided Orchestra 2 with a cut-off run and was surprised by the good-sounding filter.
The Bass Sound is a modified variant of SQR-BassPad.
The FX Sound and the E.Piano are new Performances.

**Reply by probonopd** [reply]
Would love to learn how to make music like that. I hear so many things here I would not know how to do with (Mini)Dexed. It would be very instructional to see the MIDI files, which then one would have to play through MiniDexed and then combine in the DAW.

For this to be really awesome we would need to find a way to make our Performances sendable through MIDI SysEx messages... (like is already possible with Voices)

**Reply by Kirtai** [reply]
> Resulting 2 miniDexed.

Why did it never occur to me to make more than one? :rofl: 

I love what you're doing here.

**Reply by probonopd** [reply]
Reminds me of Brian Eno, "I used to have seven of these" :+1: (DX7s, not MiniDexeds)

https://www.youtube.com/watch?v=21yu3RG80lU

**Comment by Banana71** [comment]
**String Demo: Midi File Demo inclusive Performance.ini**
TG 1-4: Strings, MIDI Channel 1
TG 5: Drum, MIDI Channel 3
TG 6 & 7: Floaters, MIDI Channel 4
TG 8 : not used, MIDI Channel 5
I tested the MIDI file with Audacity, just select the right Midi device and set the performance "String Demo" on the miniDexed

[String Demo.zip](https://github.com/probonopd/MiniDexed/files/10834575/String.Demo.zip)

https://user-images.githubusercontent.com/104296205/221433106-68d174b3-842a-4a28-a432-d410de9c433f.mp4

**Comment by Banana71** [comment]
**Rockstar Demo: Midi File Demo inclusive Performance.ini,   inspired by NEFFEX**
miniDexed DeepBass with tinnitus, no EQ, no FX, only one miniDexed 8 TG's 


[Rockstar.ini.mid.zip](https://github.com/probonopd/MiniDexed/files/10874794/Rockstar.ini.mid.zip)

https://user-images.githubusercontent.com/104296205/222521316-cab31770-f92a-4825-9db4-c4901247acfb.mp4

![Rockstar Tracks](https://user-images.githubusercontent.com/104296205/222522391-e25d2057-87da-41fa-8834-3fa5a20db093.jpg)

**Comment by Banana71** [comment]
I've created a few performances that should be helpful when listening through sound banks and creating performances.

[Quickstart-Performances.zip](https://github.com/probonopd/MiniDexed/files/11130408/Quickstart-Performances.zip)


Instructions for the Performances Single Voice, Dual Voice, Quad Voice and MultiTimbral

These Performances should make it easier for you to create your own Performances on the miniDexed.

Copy the extracted files to the SD-Card. copy the Performances 000130_MultiTimbral.ini, 000127_Single Voice.ini,
000128_Dual Voice.ini and 000129_Quad Voice.ini to the Performancefolder to the SD-Card.
Optional you can copy the 000000_Tfrack02.syx soundbank to /sysex/voice/ directory. This bank comes from the TX816 and contains a wide range of bread and butter sounds. If there is no syx file in the /sysex/voice/ folder with the numbering 000000_xxxxxxx.syx, the factory bank is taken from the DX7 Mark 1.  

Single Voice:
only TG1 is active, you can select Voice by Menu - TG1 - Voice. MIDIChannel 1; PAN center position

Dual Voice:
TG1 and TG2 are active, you can Select Voice via Menue - TG1 or TG2 - Voice. MIDIChannel 1; PAN TG1 left and TG2 right

Quad Voice:
TG1 to TG4 are active, you can Select Voice via Menue - TG1 to TG4 - Voice. MIDIChannel 1; PAN TG1&3 left and TG2&4 right

MultiTimbral:
TG1 to TG8 are active, you can Select Voice via Menue - TG1 to TG8 - Voice. MIDIChannel 1 - 8 ; all PAN are in Center


If no display is installed yet, you can set a voice with a master keyboard or a DAW via a program change command. Of course, it also works with a display.

**Comment by Banana71** [comment]
I created a new set of drum sounds. Let it Rock !!!

https://user-images.githubusercontent.com/104296205/232234501-98fbd251-ed43-4eba-aae7-45b85a461697.mp4

[000110_DrumKit2.zip](https://github.com/probonopd/MiniDexed/files/11239907/000110_DrumKit2.zip)

<html>
	


<body>


Performance | TG | Voice | Notelimit | Desciption
-- | -- | -- | -- | --
RockKit | TG1 | BassDrum | C0 – C1 | Banana71, Lars Ulrich
-- |TG2 | Snare | C#1 - E1 | Banana71
-- |TG3 | mDxHiHat | F1 – A1 | Banana71
-- |TG4 | mDxOpenHat | A#2 – B2 | Banana71
-- |TG5 | Hi-Hat 1 | C2 – C#2 | VRC-1003_A64
-- |TG6 | Tap Slap 1 | D2 – E2 | VRC-1003_A32
-- |TG7 | MuschelDrum | F2 – F#2 | Banana71
-- |TG8 | RockTom2 | G2 – C7 | Banana71 


</body>

</html>

**Comment by Banana71** [comment]
Is there a need for seasonal sounds here?
Today I did some scary things: 


https://github.com/probonopd/MiniDexed/assets/104296205/6f19ef41-cd89-489a-858d-d6351234af8f

--No effects, no EQ, just the internal reverb--
![realistic-photoplaying-on-halloween-on-the-pianocandle-light-cookies-creepy-old-house-293681035](https://github.com/probonopd/MiniDexed/assets/104296205/21e4e080-1658-47cb-b909-336cdd73fa88)

**Reply by Banana71** [reply]
An orchestral variant,......
I like Halloween, use headphones

https://github.com/probonopd/MiniDexed/assets/104296205/c0468c39-af26-4a87-8d70-b2b7112616c9


![Beauty Doll](https://github.com/probonopd/MiniDexed/assets/104296205/aaba44e8-c487-4cab-9cdc-8b5ad28a7cbf)

**Reply by Banana71** [reply]
miniDexed Sound-Demo III now on YouTube.
Processed some sounds in a video.  I created the images using Playground AI. 

https://youtu.be/LMYvjFVMQkQ?si=L0UT38NIjGhTpM2d
